fineract-provider/src/main/resources/db/changelog/tenant/changelog-tenant.xml

```diff
@@ -81,4 +81,5 @@
     <include file="parts/0059_add_spring_batch_loan_id_list_table.xml" relativeToChangelogFile="true"/>
     <include file="parts/0060_add_job_name_to_command_source.xml" relativeToChangelogFile="true"/>
     <include file="parts/0061_add_idempotency_key_to_command_source.xml" relativeToChangelogFile="true"/>
+    <include file="parts/0062_add_flag_attribute_to_loan.xml" relativeToChangelogFile="true"/>
```

Comment:
let's call it add_fraud_attribute_to_loan.xml

Reply:
Done